tests/end2end/features/open.feature

```diff
@@ -108,3 +108,37 @@ Feature: Opening pages
         When I run :quickmark-add http://localhost:(port)/data/numbers/8.txt quickmarktest
         And I run :open quickmarktest
         Then data/numbers/8.txt should be loaded
+
+    Scenario: :open with URL (explicit)
+        Given I open about:blank
+        When I set tabs -> new-tab-position-explicit to right
+        And I set tabs -> new-tab-position to left
+        And I run :tab-only
+        And I run :open -t http://localhost:(port)/data/numbers/9.txt
+        And I wait until data/numbers/9.txt is loaded
+        Then the session should look like:
+            windows:
+            - tabs:
+              - history:
+                - url: about:blank
+              - active: true
+                history:
+                - active: true
+                  url: http://localhost:*/data/numbers/9.txt
```

Comment:
You should be able to use "Then the following tabs should be open:" (see `tabs.feature`) to simplify this a bit (I probably should do the same for other tests in here...)